TradeItIosTicketSDK2/TradeItFxTradingTicketViewController.swift

```diff
@@ -66,6 +66,46 @@ class TradeItFxTradingTicketViewController: TradeItViewController, UITableViewDa
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
+
+        let isChosenAccountEnabled = self.order.linkedBrokerAccount?.isEnabled ?? false
```

Comment:
What about something like:
```swift
guard let isChosenAccountEnabled = self.order.linkedBrokerAccount?.isEnabled, isChosenAccountEnabled else {
     self.showAccountSelection()
     return
}
```
?

Reply:
Also what is the use case for this ? I thought we were already only displaying enabled accounts on the account selection screen ?

Reply:
If you go to the account selection screen from the ticket and disable the chosen account and then navigate back to the ticket, the disabled account will still be selected. This will detect that scenario.

Reply:
Changing to:
```
guard self.order.linkedBrokerAccount?.isEnabled ?? false {
    self.showAccountSelection()
    return
}
```